docs/governance/community-policies/definition-of-done.md

```diff
@@ -0,0 +1,14 @@
+# Definition of Done (DoD) for a Service
+
+Service in this context is anything that can not be further categorized into subcategories. For example _Relational Databases_ is a service where _Databases_ is considered as a higher level grouping of common features among all different types of databases.
+
+Stated below is the set of criteria that a service must meet for the team to consider it complete and ready for its consumers.
+
+- A comprehensive set of minimum viable features should be identified for the service.
```

Comment:
```suggestion
- A comprehensive set of minimum viable features has been identified for the service.
```

changing phrasing to past tense (as in, this must be true at the time it is considered done)